service/src/test/java/com.codeforcommunity/processor/ProcessorImplTest.java

```diff
@@ -1,318 +1,318 @@
-    processor.deleteNote(0);
-    assertEquals(1, mockDb.timesCalled("DELETE"));
-    assertEquals(-1, mockDb.timesCalled("SELECT"));
-    assertEquals(-1, mockDb.timesCalled("INSERT"));
-    assertEquals(-1, mockDb.timesCalled("UPDATE"));
-    String sql = mockDb.getSqlStrings().get("DELETE").get(0);
-    Object[] bindings = mockDb.getSqlBindings().get("DELETE").get(0);
-    assertEquals("delete from \"note\" where \"note\".\"id\" = ?", sql);
-    assertEquals(1, bindings.length);
-    assertEquals(0, bindings[0]);
-  }
-
-  /**
-   * Test calling select without priming mock db.
-   */
-  @Test
-  public void testUnprimedSelect() {
-    setup();
-    DSLContext ctx = mockDb.getContext();
-
-    Exception e = assertThrows(IllegalStateException.class, () ->
-      ctx.selectFrom(Tables.NOTE).where(Tables.NOTE.ID.eq(0)).fetchOneInto(Note.class));
-    assertEquals("You probably forgot to prime your "
-          + "JooqMock by calling addReturn (with one of SELECT/INSERT/UPDATE/DELETE as "
-          + "your operation.", e.getMessage());
-  }
-
-  /**
-   * Test calling insert without priming mock db.
-   */
-  @Test
-  public void testUnprimedInsert() {
-    setup();
-    DSLContext ctx = mockDb.getContext();
-
-    Exception e = assertThrows(IllegalStateException.class, () -> {
-      NoteRecord record = ctx.newRecord(Tables.NOTE);
-      record.setTitle("Hello");
-      record.setBody("World");
-      record.store();
-    });
-    assertEquals("You probably forgot to prime your "
-        + "JooqMock by calling addReturn (with one of SELECT/INSERT/UPDATE/DELETE as "
-        + "your operation.", e.getMessage());
-  }
-
-  /**
-   * Test calling update without priming mock db.
-   */
-  @Test
-  public void testUnprimedUpdate() {
-    setup();
-    DSLContext ctx = mockDb.getContext();
-
-    // prime database for SELECT
-    NoteRecord primer = new NoteRecord();
-    primer.setId(0);
-    primer.setTitle("TITLE");
-    primer.setBody("SET BODY");
-    mockDb.addReturn("SELECT", primer);
-
-    Exception e = assertThrows(IllegalStateException.class, () -> {
-      NoteRecord update = ctx.fetchOne(Tables.NOTE, Tables.NOTE.ID.eq(0));
-      update.setTitle("Hello");
-      update.store();
-    });
-    assertEquals("You probably forgot to prime your "
-        + "JooqMock by calling addReturn (with one of SELECT/INSERT/UPDATE/DELETE as "
-        + "your operation.", e.getMessage());
-  }
-
-  /**
-   * Test calling delete without priming mock db.
-   */
-  @Test
-  public void testUnprimedDelete() {
-    setup();
-    DSLContext ctx = mockDb.getContext();
-
-    Exception e = assertThrows(IllegalStateException.class, () ->
-      ctx.deleteFrom(Tables.NOTE).where(Tables.NOTE.ID.eq(0)).execute());
-    assertEquals("You probably forgot to prime your "
-        + "JooqMock by calling addReturn (with one of SELECT/INSERT/UPDATE/DELETE as "
-        + "your operation.", e.getMessage());
-  }
-}
+//package com.codeforcommunity.processor;
```

Comment:
You are not allowed to check in a commented out test file... especially one you just finished writing last sprint.

Reply:
Interesting, I don't remember when/why I did that. I just re-ran all of the tests and everything passed.